.github/workflows/c-coverage.yml

```diff
@@ -0,0 +1,56 @@
+name: C coverage
+
+on:
+  schedule:
+    # Run this daily at 2400 UTC
+    - cron: '0 0 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  c-coverage:
+    name: 'C-level coverage (using llvm-cov)'
+    runs-on: ubuntu-20.04
```

Comment:
Any reason for using 20.04 instead of 22.04?
```suggestion
    runs-on: ubuntu-22.04
```

Reply:
> (That said, we could consider upgrading all of the Ubuntu's in tandem in a separate PR, of course...)

+1